packages/host/app/components/prerendered-card-search.gts

```diff
@@ -175,6 +176,8 @@ export default class PrerenderedCardSearch extends Component<Signature> {
     let { query, format, cardUrls, realms } = this.args;
 
     let realmsChanged = !isEqual(realms, this._lastRealms);
+    // guard against a query that is deeply equal to the last query, but not strictly equal
+    let queryChanged = !isEqual(query, this._lastSearchQuery);
```

Comment:
We make a similar guard in PrerenderedSearch as well which has the same vulnerability